Libplanet/Store/LiteDBStore.cs

```diff
@@ -424,59 +353,19 @@ public override AddressStateMap GetBlockStates(HashDigest<SHA256> blockHash)
             Block<T> branchPoint,
             IImmutableSet<Address> addressesToStrip)
         {
-            long branchPointIndex = branchPoint.Index;
-            List<LiteFileInfo> files =
-                _db.FileStorage
-                    .Find($"{StateRefIdPrefix}{srcNamespace}")
-                    .ToList();
+            string srcCollId = StateRefId(@srcNamespace);
+            string dstCollId = StateRefId(@destNamespace);
```

Comment:
I think we can omit the `@` prefix. Is this intended?

Reply:
No, it's not intended.  I'll remove them.